app/components/skip_link_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<nav id="skip" aria-label="<%= t('blacklight.skip_links.label') %>">
+<nav id="skip" class="visually-hidden-focusable" aria-label="<%= t('blacklight.skip_links.label') %>">
```

Comment:
Nav should not have this class.  It belongs on the anchor tag. See https://github.com/sul-dlss/component-library/blob/main/header/index.html#L34

Reply:
Maybe this is a blacklight bug then? I think the skip to first result link is coming from blacklight: https://github.com/projectblacklight/blacklight/blob/6a5f890ccbd379085c4bd74363619650f3872764/app/views/catalog/_search_results.html.erb#L11

Reply:
Yes. Exactly.  we need to put the classes on that link.

Reply:
We can work on fixing this in Blacklight, but it seems not ideal to have this in production in exhibits until that gets resolved upstream. The other option I see is override the whole search_results partial in exhibits.

As part of the Blacklight fix should we backport https://github.com/projectblacklight/blacklight/pull/3461 to Blacklight 8?